test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -68,10 +84,19 @@ public static IEnumerable<object[]> GetVerifyImagesData()
                 versionFilterPattern = "^" + Regex.Escape(VersionFilter).Replace(@"\*", ".*").Replace(@"\?", ".") + "$";
```

Comment:
Consider moving this to a method, which accepts the string to be escaped.
```C#
"^" + Regex.Escape(VersionFilter).Replace(@"\*", ".*").Replace(@"\?", ".") + "$"
```

Reply:
Good suggestion - fixed.